### PR TITLE
Adding EnableManagedFolders flag in config file

### PR DESCRIPTION
**Pull request by Tulsishah**
### Description
Adding EnableManagedFolders flag in config file, by default it will be true and Cx can make it false with 
```
list:
   enable-managed-folders: true
```

### Link to the issue in case of a bug fix.
NA

### Testing details
1. Manual - Tested all IAM permissions scenarios.
2. Unit tests - NA
3. Integration tests - Automated

### CODE REVIEW

File: internal/config/yaml_parser_test.go
Comment (sethiay) [inline internal/config/yaml_parser_test.go:36]: please keep this check only once.

Comment (sethiay) [reply internal/config/yaml_parser_test.go:36]: my comment is not correct.

**Review by sethiay — APPROVED** [review]
Approving, however please wait for @raj-prince  approval before merging. Thanks.

**Review by raj-prince — APPROVED** [review]

**Pull request merged**
